### DISCUSSION TITLE
作业需要做到什么程度?

### DISCUSSION THREAD

**Discussion by yokooseed** [west2-online/learn-go#3] [Q&A] [answered]
如题, 很好奇作业需要做到什么程度才算合格

**Comment by ozline** [comment, marked as answer]
这取决于你对作业的认知

1. 如果你想成为工作室的正式成员, 我们建议你适当的完成一定的Bouns内容以确保你的作业完成的质和量在同组的同学中是靠前的
2. 如果你想学习, 可以只完成基础的内容, 可以联系我们给你的作业进行评价
